### DISCUSSION TITLE
how to redirect user to requested page instead of redirect url

### DISCUSSION THREAD

**Discussion by rarathore** [OpenIDC/mod_auth_openidc#1043] [Q&A]
Hi Team,

I configured Apache with Mod_OIDC. I received links in mail and when I clicked the link, it will check the session and if not exists it will redirect me to login page, after successful logged in, it will redirect me to home page instead of called URL, can you please suggest how can I overcome with this issue.

**Comment by Clivern** [comment]
Upgrade to newer versions something like 2.4.14.3. and use the following
```
OIDCRedirectURI /redirect_uri
```

Take a look at https://github.com/OpenIDC/mod_auth_openidc/discussions/875
